common/build.gradle

```diff
@@ -57,8 +67,8 @@ android {
             minifyEnabled false
             debuggable false
             testCoverageEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-            consumerProguardFiles 'consumer-rules.pro'
+            //proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            //consumerProguardFiles 'consumer-rules.pro'
```

Comment:
Why comment this out?

Reply:
Good catch.  That was an investigation and turned out to be a red herring.

Reply:
fixed